README.md

```diff
@@ -105,9 +112,7 @@ And add this to your _CMakeLists.txt_
 list(APPEND CMAKE_PREFIX_PATH "/path/to/ica/installation")
 
 find_package(ICA CONFIG REQUIRED)
-ica_set_checks(<your checks list>)
-
-target_link_libraries(<your target> ICA::ICAChecks)
+add_ica_checks(<your checks list>)
```

Comment:
As I understand, we also have `target_` alternative of this function. Maybe we want to document it too?